src/set_theory/cardinal/basic.lean

```diff
@@ -1420,6 +1425,12 @@ lemma mk_le_mk_of_subset {α} {s t : set α} (h : s ⊆ t) : #s ≤ #t :=
 lemma mk_subtype_mono {p q : α → Prop} (h : ∀ x, p x → q x) : #{x // p x} ≤ #{x // q x} :=
 ⟨embedding_of_subset _ _ h⟩
 
+lemma le_mk_sdiff_add_mk (S T : set α) : #S ≤ #(S \ T : set α) + #T :=
```

Comment:
I think `diff` rather than `sdiff` would be better here, since we use `diff` for sets

Reply:
It's a mess, really Some do and some don't (`set.definable.sdiff` for example). I prefer `sdiff` because it's the name of the general operation and how it's called on `finset`. I will open a PR to replace `diff` with `sdiff` in the `set` lemmas.